packages/framer-motion/src/gestures/hover.ts

```diff
@@ -20,7 +19,7 @@ function addHoverEvent(node: VisualElement<Element>, isActive: boolean) {
         }
 
         if (props[callbackName]) {
-            frame.update(() => props[callbackName](event, info))
+            props[callbackName](event, info)
```

Comment:
Is this an unrelated/drive-by change to no longer defer event handlers?

Reply:
It's actually entwined because without doing this we can add a two-frame delay between interactions and paint (which is counter to the point of this) as we have the frame for the event then the frame to resolve keyframes

Reply:
Ah!